server/services/caseloadService.test.ts

```diff
@@ -1485,6 +1485,23 @@ describe('Caseload Service', () => {
         },
       ] as OffenderDetail[])
 
+      prisonerService.searchPrisonersByNomisIds.mockResolvedValue([
```

Comment:
Unit test file for `caseloadService` has become quite messy, need to come back to tidy it up